src/shared.ts

```diff
@@ -326,7 +328,9 @@ export function BlockBlueVerified(user: BlueBlockerUser, headers: any, config: C
         `did not block Twitter Blue verified user ${formattedUserName} because they have over a million followers and Elon is an idiot.`,
       );
     } else {
-      queueBlockUser(user, String(user.rest_id), headers, ReasonBlueVerified);
+      let reason = ReasonBlueVerified;
+      if (hasBlockableVerifiedTypes) reason = ReasonBusinessVerified;
```

Comment:
Nit - Generally I find this style of `if` more confusing and prefer the included {} for clarity, but definitely not worth blocking over

Reply:
for sure, I generally agree, too. Trying to break that habit.